htdocs/bom/class/bom.class.php

```diff
@@ -1449,13 +1449,9 @@ public function calculateCosts()
 							$this->error = $tmpproduct->error;
 							return -1;
 						}
-<<<<<<< HEAD
-						$line->unit_cost = (float) price2num((!empty($tmpproduct->cost_price)) ? $tmpproduct->cost_price : (float) $tmpproduct->pmp);
-=======
 						$unit_cost = (!empty($tmpproduct->cost_price)) ? $tmpproduct->cost_price : $tmpproduct->pmp;
 						$line->unit_cost = (float) price2num($unit_cost);
->>>>>>> develop
-						if (empty($line->unit_cost)) {
+						if (empty($unit_cost)) {
```

Comment:
Note, this is new - I figured that with (float) price2num, $line->unit_cost can not be empty, so we need to check the temporary variable before conversion.